src/schema/nmdc.yaml

```diff
@@ -1625,9 +1625,21 @@ enums:
       Metagenome Bins:
```

Comment:
Let's make sure the intention of these two enums is clear:
- Metagenome Bins:
- Metagenome Bins File: